xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminSheet.xml

```diff
@@ -61,8 +61,8 @@
     'displayBeforeCategory': 'content',
     'children': [
        {'id' : 'Themes', 'perSpace' : true, 'order' : 100},
-       {'id' : 'Panels.PanelWizard', 'perSpace' : true, 'order' : 200},
-       {'id' : 'Presentation', 'perSpace' : true, 'order' : 300}
+       {'id' : 'Panels.PanelWizard', 'perSpace' : true, 'order' : 300},
```

Comment:
Could be good to add a comment to explain why we go from 100 to 300 directly now.

Reply:
yeap. It would be nice to comment with the missing pieces, something like: 
```
## Menu.MenuConfigurationSection (200)
{'id' : 'Panels.PanelWizard', 'perSpace' : true, 'order' : 300},
## . . 400
## ....
```
This should be documented in the code. An alternative would be to also have it somewhere on xwiki.org

Reply:
Alright.
Would this be fine?:
{'id' : 'Themes', 'perSpace' : true, 'order' : 100},
## Menus | menu.name | order: 200
{'id' : 'Panels.PanelWizard', 'perSpace' : true, 'order' : 300},
## Applications Panel | panels.applications | order: 400
## Navigation Panel | panels.navigation | order: 500
{'id' : 'Presentation', 'perSpace' : true, 'order' : 600}

Reply:
It's good that you've added the order. 
But the name "Menus", "Application Panel" could change over time, since it's a translation, so it's not static information. A Translation key maybe would be better, since that key doesn't change (or less often and only if it gets deprecated). 

What I would like is the page that contains the extension point. Like in the case of Menu, the page where you need to go in order to change the order would be "Menu.MenuConfigurationSection".

Reply:
Then how about?:
```
{'id' : 'Themes', 'perSpace' : true, 'order' : 100},
## admin.menu.name |  order: 200 | Menu.MenuConfigurationSection
{'id' : 'Panels.PanelWizard', 'perSpace' : true, 'order' : 300},
## admin.panels.applications | order: 400 | PanelsCode.ApplicationsPanelConfigurable
## admin.panels.navigation | order: 500 | PanelsCode.NavigationConfigurationSection
{'id' : 'Presentation', 'perSpace' : true, 'order' : 600}
```

Reply:
How about: 
A.
## displayInSection: menu.name | sectionOrder: 200 | page: Menu.MenuConfigurationSection

or
B.
## page: Menu.MenuConfigurationSection | order: 200

wdyt?

Reply:
'**A**' looks good since it gives all the required info.